server/interceptor/interceptor.go

```diff
@@ -258,6 +274,15 @@ func isLeafOK(leaf *trillian.QueuedLogLeaf) bool {
 	return leaf == nil || leaf.Status == nil || leaf.Status.Code == int32(codes.OK)
 }
 
+// serviceName returns "some.package.service" for
+// "/some.package.service/method".
+func serviceName(fullMethod string) string {
+	if !strings.HasPrefix(fullMethod, "/") {
+		return ""
+	}
+	return strings.Split(fullMethod, "/")[1]
```

Comment:
This should be ...[0], shouldn't it?

Reply:
[0] gets us "" because strings start with "/"

Reply:
Ah, bad assumption on my part. Carry on!